packages/calypso-products/test/is-jetpack-legacy-item.js

```diff
@@ -15,8 +17,10 @@ describe( 'isJetpackLegacyItem', () => {
 	} );
 
 	it( 'should return false otherwise', () => {
+		expect( isJetpackLegacyItem( PRODUCT_JETPACK_ANTI_SPAM_BI_YEARLY ) ).toEqual( false );
 		expect( isJetpackLegacyItem( PRODUCT_JETPACK_ANTI_SPAM ) ).toEqual( false );
 		expect( isJetpackLegacyItem( PRODUCT_JETPACK_ANTI_SPAM_MONTHLY ) ).toEqual( false );
+		expect( isJetpackLegacyItem( PLAN_JETPACK_COMPLETE_BI_YEARLY ) ).toEqual( false );
```

Comment:
Nothing critical but there might be some better words to use for multi year plans :)

https://github.com/Automattic/wp-calypso/blob/370b3b8ef2426b65732a81fc4a5d68dae257df6c/packages/calypso-products/src/constants/terms.ts#L27-L37

Reply:
In this case I was using the same naming convention set by the previous Jetpack 2 year plans `JETPACK_SECURITY_T1_BI_YEARLY` and `JETPACK_BACKUP_T1_BI_YEARLY`

Thank you for pointing this out though and I will keep it in mind in the future 😄